wordpress-mobile/WordPress-iOS
# Rich Text: Do not set frame if it has an infinite origin.

Fixes #6276 
When a 180 degree rotation occurs, `frameForTextInRange` will return a rect with an infinite origin, and cgsize zero.  This pr checks for such a frame and avoids setting it on attachment views.

To test:
Set a breakpoint on the return so its easier to test the fix.
Repeat the steps outlined in the issue and confirm that images do not vanish when a 180 degree rotation occurs.

Needs review: @frosty
